### PR TITLE
feat: mana_audio

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

there ya go

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Review by Manason — APPROVED** [review]

**Review by D4isDAVID — DISMISSED** [review]
[Manason/mana_audio releases](https://github.com/Manason/mana_audio/releases) do not seem to provide any file named `mana_audio.zip`. If we *will* add this in the future, Manason has to either provide such a file, or we have to download the source code zip similarly to ox_fuel.

**Comment by Manason** [comment]
> [Manason/mana_audio releases](https://github.com/Manason/mana_audio/releases) do not seem to provide any file named `mana_audio.zip`. If we _will_ add this in the future, Manason has to either provide such a file, or we have to download the source code zip similarly to ox_fuel.

Yeah, I'm trying to cut a new release with the Qbox release scripts, but running into some errors. Will get out a new release shortly.

File: qbox-lean.yaml [hunk 1]
```diff
@@ -115,6 +115,13 @@ tasks:
     dest: ./resources/[standalone]/loadscreen/html/assets/logo.png
     overwrite: true
 
+  - action: download_file
+    path: ./tmp/mana_audio.zip
+    url: https://github.com/Manason/mana_audio/releases/latest/download/mana_audio.zip
+  - action: unzip
+    dest: ./resources/[qbx]
```
Comment (solareon) [inline qbox-lean.yaml:122]:
```suggestion
    dest: ./resources/[standalone]
```

File: qbox-stable.yaml [hunk 2]
```diff
@@ -112,6 +112,13 @@ tasks:
     dest: ./resources/[standalone]/loadscreen/html/assets/logo.png
     overwrite: true
 
+  - action: download_file
+    path: ./tmp/mana_audio.zip
+    url: https://github.com/Manason/mana_audio/releases/latest/download/mana_audio.zip
+  - action: unzip
+    dest: ./resources/[qbx]
```
Comment (solareon) [inline qbox-stable.yaml:119]:
```suggestion
    dest: ./resources/[standalone]
```

File: qbox.yaml [hunk 3]
```diff
@@ -138,6 +138,13 @@ tasks:
     dest: ./resources/[standalone]/loadscreen/html/assets/logo.png
     overwrite: true
 
+  - action: download_file
+    path: ./tmp/mana_audio.zip
+    url: https://github.com/Manason/mana_audio/releases/latest/download/mana_audio.zip
+  - action: unzip
+    dest: ./resources/[qbx]
```
Comment (solareon) [inline qbox.yaml:145]:
```suggestion
    dest: ./resources/[standalone]
```

**Review by solareon — CHANGES_REQUESTED** [review]

**Review by solareon — APPROVED** [review]

**Pull request merged**